client/src/pages/AdminHomepage.tsx

```diff
@@ -1,12 +1,15 @@
 import React, { FunctionComponent } from 'react';
+import AdminRequestGroupList from '../components/organisms/AdminRequestGroupList';
+import Navbar from '../components/organisms/Navbar';
 
 const AdminHomepage: FunctionComponent = () => {
-
-  return (
-    <div>
-        I am the admin homepage
-    </div>
-  );
+    return (
+        <div>
+            <Navbar links={[{ name: "Donation Hub", link: "/donation-guidelines" },
```

Comment:
TODO: Use AdminPageLayout in https://github.com/uwblueprint/pregnancy-centre/pull/99

Reply:
when you say TODO, does that mean as part of this PR or as a followup?

Reply:
We just need #99 to be merged. We can do this before/after merging this.